Assets/MixedRealityToolkit/UX/Scripts/Tooltip/ToolTipBackgroundBlob.cs

```diff
@@ -0,0 +1,200 @@
+        public Transform RotationTarget;
+        public Transform DistortionTarget;
+        public Transform AttachPointOffset;
+
+        public float BlobInertia {
+            get {
+                return blobInertia;
+            }
+            set {
+                blobInertia = Mathf.Clamp(value, 0, MaxInertia);
+            }
+        }
+
+        public float BlobDistortion {
+            get {
+                return blobDistortion;
+            }
+            set {
+                blobDistortion = Mathf.Clamp(value, 0, MaxDistortion);
+            }
+        }
+
+        public float BlobRotation {
+            get {
+                return blobRotation;
+            }
+            set {
+                blobRotation = Mathf.Clamp(value, 0, MaxRotation);
+            }
+        }
+
+        public float PositionCorrectionStrength {
+            get {
+                return positionCorrectionStrength;
+            }
+            set {
+                positionCorrectionStrength = Mathf.Clamp(value, MinPositionCorrection, MaxPositionCorrection);
+            }
+        }
+
+        public float DistortionCorrectionStrength {
+            get {
+                return distortionCorrectionStrength;
+            }
+            set {
+                distortionCorrectionStrength = Mathf.Clamp(value, MinDistortionCorrection, MaxDistortionCorrection);
+            }
+        }
+
+        public float RotationCorrectionStrength {
+            get {
+                return rotationCorrectionStrength;
+            }
+            set {
+                rotationCorrectionStrength = Mathf.Clamp (value, MinRotationCorrection, MaxRotationCorrection);
+            }
+        }
+
+        public Vector3 BlobOffset {
+            get {
+                return blobOffset;
+            } set {
+                blobOffset = value;
+            }
+        }
+
+        /// <summary>
+        /// Mesh renderer for mesh background.
+        /// </summary>
+        public MeshRenderer BackgroundRenderer;
+
+        [Header("Blob settings")]
+        [SerializeField]
+        [Range(0f,5f)]
+        private float blobInertia = 0.5f;
+
+        [SerializeField]
+        [Range(0f, 1f)]
+        private float blobDistortion = 0.1f;
+
+        [SerializeField]
+        [Range(0f, 1f)]
+        private float blobRotation = 0.1f;
+
+        [SerializeField]
+        [Range(0.1f, 5f)]
+        private float positionCorrectionStrength = 1f;
+
+        [SerializeField]
+        [Range(0.1f, 5f)]
+        private float distortionCorrectionStrength = 1f;
+
+        [SerializeField]
+        [Range(0.1f, 5f)]
+        private float rotationCorrectionStrength = 1f;
+
+        [SerializeField]
+        private Vector3 blobOffset;
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+            lastPosition = PositionTarget.position;
+            inertialContentBounds = new Bounds(Vector3.zero, Vector3.one);
```

Comment:
this default bounds could be declared in a private field and reused each time it's needed to reset values.